src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MediaPlayerElement.cs

```diff
@@ -11,38 +16,577 @@
 
 using _MediaPlayer = Windows.Media.Playback.MediaPlayer; // alias to avoid same name root namespace from ios/macos
 
-namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls;
+#if __WASM__
+[Ignore("UNO TODO - This test is failing on WASM [https://github.com/unoplatform/uno/issues/12665]")]
+#endif
+[TestClass]
+[RunsOnUIThread]
+public partial class Given_MediaPlayerElement
 {
-	[TestClass]
-	[RunsOnUIThread]
-	public partial class Given_MediaPlayerElement
+	private static readonly Uri TestVideoUrl = new Uri("https://test-videos.co.uk/vids/bigbuckbunny/mp4/h264/720/Big_Buck_Bunny_720_10s_5MB.mp4");
+
+	[TestMethod]
+	public async Task When_MediaPlayerElement_NotAutoPlay_Source()
```

Comment:
It seems to be running fine on CI indeed!

![image](https://github.com/unoplatform/uno/assets/5839577/8630454a-3930-4099-afec-c816d00d0d6c)